packages/effects/access/src/accessible.ts

```
@@ -42,16 +42,16 @@ async function generateAccessible(
         delete route.component;
       }
       // 根据router name判断，如果路由已经存在，则不再添加
-      if (!names?.includes(route.name)) {
-        root.children?.push(route);
-      } else {
+      if (names?.includes(route.name)) {
         // 找到已存在的路由索引并更新，不更新会造成切换用户时，一级目录未更新，homePath 在二级目录导致的404问题
         const index = root.children?.findIndex(
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)